boss-ws/src/main/java/org/genomebridge/boss/http/service/BossAPI.java

```diff
@@ -14,10 +15,15 @@
 
     @JsonInclude(Include.NON_NULL)
     public static class ObjectDesc extends ObjectCore {
+
         public Boolean forceLocation;
-        public String[] readers, writers;
+        @ApiModelProperty(value = "this field lists the (BOSS) users who have read access to this BOSS object")
```

Comment:
It would be nice to have a dataType for this property and the next one as well:
http://docs.swagger.io/swagger-core/apidocs/com/wordnik/swagger/annotations/ApiModelProperty.html#dataType()